contracts/interfaces/ICover.sol

```diff
@@ -9,27 +9,44 @@ interface ICover is IERC721 {
   /* ========== DATA STRUCTURES ========== */
 
   struct StakingPool {
-    uint160 id;
-    uint96 bookedAmount;
+    address poolAddress;
+    uint96 coverAmount;
+    uint96 premiumInNXM;
   }
 
   struct Cover {
```

Comment:
naming clash with the contract, we should rename one of them

Reply:
CoverData ?